src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -312,15 +312,15 @@ object Task {
   object Id {
 
     object Names {
-      val anonymousContainer = "$anon" // presence of `$` is important since it's illegal for a real container name!
+      val anonymousContainer = "_app" // presence of `_` is important since it's illegal for a real container name!
```

Comment:
can we go with appContainer name here instead of anonymous? Or at least put into comments that this is mainly for apps?